base/irrationals.jl

```diff
@@ -2,6 +2,11 @@
 
 ## general machinery for irrational mathematical constants
 
+"""
+    Irrational <: Real
+
+Abstract supertype for irrational numbers.
```

Comment:
Yea, kinda acts like a supertype though. I will move it and update the docstring.